sscanss/app/window/theme_manager.py

```diff
@@ -0,0 +1,131 @@
+import sys
+from PyQt6 import QtCore, QtGui, QtWidgets
+from sscanss.config import settings, Key, Themes, load_stylesheet
+
+
+class ThemeManager(QtWidgets.QWidget):
+    """Manages light and dark themes using qt properties to get colours
+    from stylesheet.
+
+    :param parent: main window instance
+    :type parent: MainWindow
+    """
+    theme_changed = QtCore.pyqtSignal()
+
+    def __init__(self, parent):
+        super().__init__(parent)
+
+        self.parent = parent
+        self.setObjectName('ColourPalette')
+        self.loadStyleSheet()
+
+    def reset(self):
+        self._html_anchor = QtGui.QColor()
+        self._scene_anchor = QtGui.QColor()
+        self._scene_path = QtGui.QColor()
+        self._scene_highlight = QtGui.QColor()
+        self._scene_bounds = QtGui.QColor()
+        self._scene_grid = QtGui.QColor()
+        self._curve_face = QtGui.QColor()
+        self._curve_line = QtGui.QColor()
+        self._curve_label = QtGui.QColor()
+        self._curve_plot = QtGui.QColor()
+
+    def loadStyleSheet(self):
+        """Loads the stylesheet for the last active theme"""
+        self.reset()
+        if settings.value(Key.Theme) == Themes.Light.value:
+            if sys.platform == 'darwin':
+                style = load_stylesheet("mac_style.css")
```

Comment:
Do you think it would be a good idea to move the `load_stylesheet` function from `config.py` to here so that all the style/theme methods are in one file ? 

It can be defined as a static method and used in the editor to load the style sheet from here.

Reply:
I had a similar idea but the `load_stylesheet` function is also used in editor so I plan to move the ThemeManager out of the app folder in another pull request